libraries/aws_cloudtrail_trails.rb

```diff
@@ -9,37 +9,39 @@ class AwsCloudTrailTrails < Inspec.resource(1)
     end
   '
 
```

Comment:
should probably add `supports platform: 'aws'` here also (and any other resources that are missing it)

Reply:
Yikes, thanks for the reminder.  Will do.